modules/rules_haskell/0.18/MODULE.bazel

```diff
@@ -1,6 +1,7 @@
 module(
     name = "rules_haskell",
     version = "0.18",
+    bazel_compatibility = [">=6.0.0", "<=6.99.0"],
```

Comment:
The design doc just mentioned three possible comparison operators, but I see that you also can use `<`.

Reply:
I did remove the bazel_compatibility setting again, it should be applied to rules_sh@0.3.0 IMO